pages/resources/links.md

```diff
@@ -10,12 +10,14 @@ subtitle: Community links and resources for RSEs
 
 _These resources can help to improve your knowledge about software engineering, and research practices._
 
- - [Better Scientific Software](https://bssw.io/): initiative to address challenges in software productivity, quality, and sustainability.
- - [Ask Cyberinfrastructure](https://ask.cyberinfrastructure.org/) Community forum for Q&A about software, infrastructure, and related topics. 
- - [Ideas Productivity](https://ideas-productivity.org/): Interoperable Design of Extreme-scale Application Software, including [guides](https://ideas-productivity.org/resources/howtos/) and [webinars](https://ideas-productivity.org/events/hpc-best-practices-webinars/)
+ - [Better Scientific Software (BSSw)](https://bssw.io/): A central hub for the community to address challenges in software productivity, quality, and sustainability
+ - [Ask Cyberinfrastructure](https://ask.cyberinfrastructure.org/): A community forum for Q&A about software, infrastructure, and related topics
+ - [IDEAS Productivity](https://ideas-productivity.org/): Interoperable Design of Extreme-scale Application Software, including [guides](https://ideas-productivity.org/resources/howtos/) and [webinars](https://ideas-productivity.org/events/hpc-best-practices-webinars/)
  - [Software Engineering for Science](https://se4science.org/)
  - [Software Engineering Body of Knowledge](https://www.computer.org/education/bodies-of-knowledge/software-engineering)
-
+ - [Software Sustainability Institute (SSI)](https://www.software.ac.uk): The (UK) SSI cultivates better, more sustainable, research software to enable world-class research; it helps people build better software, and it works with researchers, developers, funders and infrastructure providers to identify key issues and best practice in scientific software
+ - [US Research Software Sustainability Institute (URSSI)](http://urssi.us): a planning activity to build an US institute for software expertise as well as a social infrastructure that promotes an inclusive and diverse community of research software engineers, maintainers, contributors, and users, with the mission of improving the recognition, development, and use of software for a more sustainable research enterprise, through collaboratively developing education, outreach, and software services that emphasize open, transparent reproducible, and cooperative practices
```

Comment:
```suggestion
 - [US Research Software Sustainability Institute (URSSI)](http://urssi.us): a planning activity to build a US institute for software expertise as well as a social infrastructure that promotes an inclusive and diverse community of research software engineers, maintainers, contributors, and users, with the mission of improving the recognition, development, and use of software for a more sustainable research enterprise, through collaboratively developing education, outreach, and software services that emphasize open, transparent reproducible, and cooperative practices
```
That's a super long sentence too, but it's probably okay for a bullet point :)

Reply:
And note that my one change for the above is changing "an" to "a" in the second line (it doesn't seem to show up nicely).